src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -84,6 +85,9 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                 }
 
                 LogTaskToAudit(debug, result.Index, result.Configuration);
+
+                if (logAudit != null && LoggingSource.AuditLog.IsInfoEnabled)
+                    LogAuditFor(Database.Name, "DB Config", logAudit(result.Index, result.Configuration));
```

Comment:
we already have LogTaskToAudit, why not to use it?

Reply:
Done